Add object-level image processing and multi-table handling through iceberg (#424)
cytomining/CytoTable: docs/source/overview.md

  Parquet data destination type may be specified by using :code:`convert(..., dest_datatype="parquet", ...)` (:mod:`convert() <cytotable.convert.convert>`).
```

- __Apache Iceberg warehouse__: Iceberg is a way to organize a warehouse
  directory that contains multiple related tables, along with metadata that
  tracks how those tables change over time so reads stay consistent.

>   In CytoTable, Iceberg is exposed as an optional local warehouse backend which

Reviewer: ```suggestion
  CytoTable uses Iceberg as an optional local warehouse backend which
```

Check that I didn't remove/change meaning
